.travis.yml

```diff
@@ -8,20 +8,87 @@ jobs:
       env: TOXENV=py27-master
       after_success: codecov
     - if: branch = master
+      python: 3.7
```

Comment:
you can skip it, on the top of the file you have a default python version that would be used